wgpu-hal/src/dx12/suballocation.rs

```diff
@@ -63,6 +68,13 @@ mod allocation {
     ) -> Result<(HRESULT, Option<AllocationWrapper>), crate::DeviceError> {
         let is_cpu_read = desc.usage.contains(crate::BufferUses::MAP_READ);
         let is_cpu_write = desc.usage.contains(crate::BufferUses::MAP_WRITE);
+
+        // It's a workaround for Intel Xe drivers.
+        if !device.private_caps.suballocation_supported {
+            return super::committed::create_buffer_resource(device, desc, raw_desc, resource)
+                .map(|(hr, _)| (hr, None));
```

Comment:
Is mapping this necessary? Committed creates will never return an allocation.

Reply:
Because the allocation type is different.

Reply:
Makes sense. I don't think we can fix that until Mozilla finally gets around to allowing `windows-rs`.